### ISSUE TITLE
False positive for "Found sensor change, start, or stop after transmitter start date. Stopping Sensor Session."

### ISSUE THREAD

**Issue by PieterGit** [xdrip-js/Lookout#70]
I'm seeing

```
 transmitterIO:debug SyncNS Rig sensor start - date: 2018-12-29T18:52:15+01:00 +28s
  transmitterIO:debug SyncNS Rig sensor stop - date: 2018-12-30T02:00:28+01:00 +1ms
  transmitterIO:log Found sensor change, start, or stop after transmitter start date. Stopping Sensor Session. +9ms
  transmitterIO:debug Session Start: Sat Dec 29 2018 18:52:15 GMT+0100 sensorStart: Sat Dec 29 2018 18:52:15 GMT+0100 sensorStop: Sun Dec 30 2018 02:00:28 GMT+0100 +6ms
```
in the console. I believe in this case a `SensorStop` should not be sent. I also think you should not sent a `SensorStop` if there is a `CalibrateSensor` (or some other action) pending. The sensorstart is two hours in the past.

```
# lookout status

Press Ctrl-C to Exit
   Transmitter ID:  123456
    Fake Meter ID:  000000
          Pending:  [ { date: 1546124156463, type: 'StopSensor' } ]
          glucose: 208
            noise: 0
      noise index: 1
        inSession: true
     sensor state: OK
transmitter state: OK
         readDate: 2018-12-30T01:55:52+01:00
    session start: 2018-12-29T18:52:15+01:00
      session age: 0 days 7 hours 6 minutes
transmitter start: 2018-12-29T15:45:52+01:00
  transmitter age: 0 days 10 hours 13 minutes
        voltage a: 314
        voltage b: 306
      temperature: 40
       resistance: 436

Press Ctrl-C to Exit
=====================================
          Pending:  [ { date: 1546124156463, type: 'StopSensor' } ]
          Pending:  [ { date: 1546124156463, type: 'StopSensor' },
  { date: 1546124428068, type: 'StopSensor' } ]
          Pending:  []
          Pending:  [ { date: 1546124456163, type: 'StopSensor' } ]
          glucose: 207
            noise: 0
      noise index: 1
        inSession: true
     sensor state: OK
transmitter state: OK
         readDate: 2018-12-30T02:00:52+01:00
    session start: 2018-12-29T18:52:15+01:00
      session age: 0 days 7 hours 8 minutes
transmitter start: 2018-12-29T15:45:52+01:00
  transmitter age: 0 days 10 hours 15 minutes
        voltage a: 314
        voltage b: 306
      temperature: 40
       resistance: 436
```

I temporary fixed this by changing this in `transmitterIO.js`
```
  if (txmitterInSession && (sensorStartDelta > 0 || sensorStopDelta > 0)) {
      // give a 2 hour play between the sensor insert record
      // and the session start date from the transmitter
      log('Found sensor change, start, or stop after transmitter start date. Stopping Sensor Session.');
      debug(`Session Start: ${sessionStart} sensorStart: ${sensorInsert} sensorStop: ${sensorStop}`);
      //stopTransmitterSession();
      //await stopSensorSession();
    } 
```

That will remove the pending `StopSensor` messages.

**Comment by jpcunningh** [comment]
@PieterGit, do you agree this is fixed with #82 such that we can close this issue?

**Comment by PieterGit** [comment]
@jpcunningh I think it's fixed with #82 . Since I don't use Lookout anymore. I don't have problems with closing this issue.
